File: 프로그래머스/0/181932. 코드 처리하기/코드 처리하기.js
function solution(code) {
  let answer = '';
  let mode = 0;

  for (let i = 0; i < code.length; i++) {
    if (code[i] === '1') {
      mode = (mode + 1) % 2;
    } else if (mode === 0 && i % 2 === 0) {
      answer += code[i];
    } else if (mode === 1 && i % 2 === 1) {
      answer += code[i];
    }
  }

  if (answer !== '') {
    return answer;
  } else {
    return 'EMPTY';
  }
}

File: 프로그래머스/0/181932. 코드 처리하기/코드 처리하기.py
def solution(code):
    answer = []
    mode = 0

    for i, char in enumerate(code):
        if char == '1':
            mode = 1 - mode
        else:
            if mode == 0 and i % 2 == 0:
                answer.append(char)
            elif mode == 1 and i % 2 == 1:
                answer.append(char)

    if answer:
        return "".join(answer)
    else:
        return "EMPTY"